charts/datadog/templates/system-probe-configmap.yaml

```diff
@@ -73,7 +73,7 @@ data:
         anomaly_detection:
           enabled: {{ $.Values.datadog.securityAgent.runtime.securityProfile.anomalyDetection.enabled }}
         auto_suppression:
-          enabled: false
+          enabled: {{ $.Values.datadog.securityAgent.runtime.securityProfile.autoSuppression.enabled }}
```

Comment:
is it ok if the user pinned the Agent version to 7.50.x but get the updated version of the chart?

Reply:
Yes it will cause no issue